packages/contracts/src/extensions/UniswapExtension.sol

```diff
@@ -8,28 +8,36 @@ import {EmailWalletCore} from "../EmailWalletCore.sol";
 import {TokenRegistry} from "../utils/TokenRegistry.sol";
 import "../interfaces/Types.sol";
 
+import "./PoolFinder.sol";
+
+
 contract UniswapExtension is Extension {
     EmailWalletCore public core;
     ISwapRouter public router;
     TokenRegistry public tokenRegistry;
+    PoolFinder public poolFinder;
 
     // For this example, we will set the pool fee to 0.3%.
     uint24 public constant poolFee = 3000;
+    // For this example, we will set the maximum slippage to 0.5%.
```

Comment:
Minor thing; why is the comment saying "example"? This is a real contract. We can say "default slippage"

Reply:
Thank you for reviewing. Yes I think so too. Let me change this comment another PR.